### DISCUSSION TITLE
Adding `environment-ember-loose` to my `global.d.ts` file breaks the `TemplateFactory` import

### DISCUSSION THREAD

**Discussion by adc-mhaugen** [typed-ember/glint#653] [Q&A]
I have an example Ember 4.12 repo here: https://github.com/adc-mhaugen/generic-args.

When adding glint to this project the `TemplateFactory` import gives TS errors as soon as I add the `environment-ember-loose` import:

![Screenshot 2023-11-14 at 2 25 15 PM](https://github.com/typed-ember/glint/assets/14300009/c84f1584-91ce-4732-96f4-5984ed5e06c7)

Do I need to do that in some other glint friendly way?

Thanks!

**Comment by dfreeman** [comment]
Generally speaking, TypeScript doesn't like when you mix and match `import` statements at the top level of a module with `import` statements within a `declare module` block in that same module.

That said, in this specific case, that `declare module` block should be unnecessary with Glint anyway, so you should be able to just delete it. If for some reason you want to keep it around, I'd recommend moving the `import '@glint/environment-ember-loose';` statement to another module (or creating something fresh like `types/glint.d.ts` and putting it there)

**Reply by adc-mhaugen** [reply]
Thanks @dfreeman, I didn't realize I didn't need that anymore.  I'll get rid of it.
